src/Donations/resources/utils.ts

```diff
@@ -1,4 +1,5 @@
 import {useEntityRecord} from '@wordpress/core-data';
+import {EntityRecordResolution} from '@wordpress/core-data/build-types/hooks/use-entity-record';
```

Comment:
See comment here about this https://github.com/impress-org/givewp/pull/8035#discussion_r2256940078

Reply:
Resolved: https://github.com/impress-org/givewp/pull/8053